qdrant-landing/content/documentation/cloud/getting-started.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Getting Started
+weight: 1
+---
+
+# Getting Started with Qdrant Managed Cloud
+
+Welcome to Qdrant Managed Cloud! This document contains all the information you need to get started.
+
+## Prerequisites
+
+Before creating a cluster, make sure you have a Qdrant Cloud account. Detailed instructions for signing up can be found in the [Qdrant Cloud Setup](/documentation/cloud/qdrant-cloud-setup/) guide. You also need to provide [payment details](/documentation/cloud/pricing-payments/). If you have a custom payment agreement, first create your account, then [contact our Support Team](https://support.qdrant.io/) to finalize the setup.
+
+Premium Plan subscribers can enable single sign-on (SSO) for their organizations. To activate SSO, please reach out to the Support Team at [https://support.qdrant.io/](https://support.qdrant.io/) for guidance.
+
+## Cluster Sizing
+
+Before deploying any cluster, consider the resources needed for your specific workload. Our [Capacity Planning guide](/documentation/guides/capacity-planning/) describes how to assess the required CPU, memory, and storage. Additionally, the [Pricing Calculator](https://cloud.qdrant.io/calculator) helps you estimate associated costs based on your projected usage.
+
+## Creating and Managing Clusters
+
+After setting up your account, you can create a Qdrant Cluster by following the steps in [Create a Cluster](/documentation/cloud/create-cluster/).
+
+**Please Note:** In Hybrid Cloud environments, Qdrant clusters remain internal to your Kubernetes cluster by default. To connect to your database from outside the Kubernetes environment, refer to the [Authentication documentation](/documentation/cloud/authentication/).
```

Comment:
I'd remove this paragraph. So far, this is a getting started for managed cloud. Hybrid Cloud has some difference, e.g. you have to create an environment first. We could handle this in a dedicated hybrid cloud getting started at some point.

Also the link is wrong, it would need to point to the hybrid cloud docs page.